### PR TITLE
gnome-extra/gnome-system-monitor: Version bump to 46.0, gnome-base/libgtop: Version bump to 2.41.3

### PR DESCRIPTION
**Pull request by flint2**

### CODE REVIEW

File: gnome-extra/gnome-system-monitor/gnome-system-monitor-46.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="The Gnome System Monitor"
+HOMEPAGE="https://help.gnome.org/users/gnome-system-monitor/stable/"
```
[the remark this answers is not in the document]
Comment (flint2) [reply gnome-extra/gnome-system-monitor/gnome-system-monitor-46.0.ebuild]: Done in https://github.com/gentoo/gentoo/pull/34678/commits/6365b37996c344e188a29cdaa1649a6ec1361299

File: gnome-base/libgtop/libgtop-2.41.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
[the remark this answers is not in the document]
Comment (flint2) [reply gnome-base/libgtop/libgtop-2.41.3.ebuild]: Done in https://github.com/gentoo/gentoo/pull/34678/commits/486f1aaf14e4239862ab5f8dc4311896cedc1e7a

File: gnome-base/libgtop/libgtop-2.41.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	introspection? ( >=dev-libs/gobject-introspection-0.6.7:= )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	>=dev-build/gtk-doc-am-1.4
+	>=sys-devel/gettext-0.19.6
+	virtual/pkgconfig
+"
+
+PATCHES=(
```
[the remark this answers is not in the document]
Comment (flint2) [reply gnome-base/libgtop/libgtop-2.41.3.ebuild]: Done in https://github.com/gentoo/gentoo/pull/34678/commits/486f1aaf14e4239862ab5f8dc4311896cedc1e7a

File: gnome-base/libgtop/libgtop-2.41.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	#"${WORKDIR}"/patches # patches from master (not stable branch) that seem safe and fix potential eautoreconf problems
+	#"${FILESDIR}"/2.40.0-sandbox-workaround.patch # requires suid handling in ebuild - https://gitlab.gnome.org/GNOME/libgtop/issues/48
+	#"${FILESDIR}"/2.40.0-clang.patch
+)
+
+src_configure() {
+	# Add explicit stdc, bug #628256
+	append-cflags "-std=c99"
```
[the remark this answers is not in the document]
Comment (flint2) [reply gnome-base/libgtop/libgtop-2.41.3.ebuild]: Done in https://github.com/gentoo/gentoo/pull/34678/commits/486f1aaf14e4239862ab5f8dc4311896cedc1e7a

File: gnome-base/libgtop/libgtop-2.41.3.ebuild [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+	gnome2_src_configure \
+		--disable-static \
+		$(use_enable introspection)
+}
+
+src_install() {
+	gnome2_src_install
+	#chmod 4755 "${ED}"/usr/bin/libgtop_server2 || die
```
[the remark this answers is not in the document]
Comment (flint2) [reply gnome-base/libgtop/libgtop-2.41.3.ebuild]: Done in https://github.com/gentoo/gentoo/pull/34678/commits/486f1aaf14e4239862ab5f8dc4311896cedc1e7a

File: gnome-extra/gnome-system-monitor/gnome-system-monitor-46.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+RDEPEND="
+	>=dev-cpp/glibmm-2.68:2.68
+	>=dev-libs/glib-2.56.0:2
+	>=gui-libs/gtk-4.12.0:4[X(+)?]
+	>=dev-cpp/gtkmm-4.0.0:4.0
+	>=gnome-base/libgtop-2.41.2:2=
+	>=gnome-base/librsvg-2.46:2
+	>=dev-libs/libxml2-2.0:2
+	X? ( >=x11-libs/libwnck-2.91.0:3 )
```
Comment (eli-schwartz) [inline gnome-extra/gnome-system-monitor/gnome-system-monitor-46.0.ebuild:23]: This dependency was removed upstream... https://gitlab.gnome.org/GNOME/gnome-system-monitor/-/commit/ca3299b64522d2ef5f5bce7b7e96f58a07fa4dfd

**Pull request closed without merging**